eventstore/datadog_checks/eventstore/data/eventstore.yaml.default

```diff
@@ -0,0 +1,342 @@
+init_config:
```

Comment:
we're deprecating `init_config`, I'd move `default_timeout` and `tag_by_url` down at the `instance` level.
Regarding the `metric_definitions`, those should moved at the instance level as well, I'm assuming the list here contains the default metrics we always want to grab? In that case we could move their definition from the config file to the code.

Reply:
I will look at doing this,
Metric Definitions are a global setting, it is giving the check knowledge of all the metrics, and means that if new metrics appear it would be easy to add them to the check without needing to fully understand the code.
I can add them to the code but it feels like it defeats the point here as I would like them globally accessible and easily changeable, do you have any other suggestions (Metrics seems too repetitive)

Reply:
We could use the same approach as the JMX-fetch based checks, let's say the `kafka` check.
The default set of metrics is kept in a special config file named `metrics.yaml`, see here: https://github.com/DataDog/integrations-core/blob/master/kafka/datadog_checks/kafka/data/metrics.yaml

Then, in the regular config file, one can specify that they want whatever metric is listed in `metrics.yaml` without any further configuration step: https://github.com/DataDog/integrations-core/blob/master/kafka/datadog_checks/kafka/data/conf.yaml.example#L62
or they could list explicitly what specific metric they want to collect.

The only difference would be keeping all of this at the `instance` level, for backward compatibility reasons we couldn't move metrics out of `init_config` for the kafka check.